src/miner.erl

```diff
@@ -414,7 +451,8 @@ handle_call({create_block, Metadata, Txns, HBBFTRound}, _From, State) ->
                                election_epoch => ElectionEpoch,
                                epoch_start => EpochStart,
                                seen_votes => SeenVectors,
-                               bba_completion => BBA
+                               bba_completion => BBA,
+                               snapshot_hash => SnapshotHash
```

Comment:
The empty binary is the PBC default value, right?

Reply:
yes, this should cause nothing to be put into the proto for a bytes field, but we should make sure that a mixed cluster can make progress.

Reply:
This has been confirmed